streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -410,8 +414,10 @@ public void pipeInput(final List<ConsumerRecord<byte[], byte[]>> records) {
      */
     public void advanceWallClockTime(final long advanceMs) {
         mockTime.sleep(advanceMs);
-        task.maybePunctuateSystemTime();
-        task.commit();
+        if (task != null) {
```

Comment:
Could a task be `null` here?

Reply:
In theory yes, if you only add global stores... (better to have it, even if is should not really happen in practice)